helpdesk/tests/test_get_email.py

```diff
@@ -20,35 +20,114 @@
     # Python < 3.3
     import mock
 
+unrouted_socks_server = "127.0.0.1"
```

Comment:
Ok, ideally this would be the way to go. Be advised, I ran into some weird deviation from the expected testing behavior when using non-existing ports or nonsensical ips -- some errors that are raised differ from the ones I tried to catch